bats/core/api/ln-receive.bats

```diff
@@ -0,0 +1,380 @@
+#!/usr/bin/env bats
+
+load "../../helpers/cli.bash"
+load "../../helpers/user.bash"
+load "../../helpers/onchain.bash"
+load "../../helpers/ln.bash"
+load "../../helpers/wallet.bash"
+load "../../helpers/callback.bash"
+load "../../helpers/subscriber.bash"
+
+setup_file() {
+  lnds_init
```

Comment:
Initialising the lightning setup should be done as part of tilt up IMO

Reply:
#3718